### PR TITLE
BruteForceBlocker feed integration

### PR DESCRIPTION
**Pull request by roysagi**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
BruteForceBlocker feed

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [x] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/FeedBruteForceBlocker/Integrations/FeedBruteForceBlocker/FeedBruteForceBlocker.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    params['indicator_type'] = FeedIndicatorType.IP
+
+    params['url'] = 'http://danger.rulez.sk/projects/bruteforceblocker/blist.php'
+    params['ignore_regex'] = "^#.*"
+    params['indicator'] = json.dumps({
+        "regex": r"^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}",
+    })
+
+    fields = json.dumps({
+        "upatedate": {
```
Comment (avidan-H) [inline Packs/FeedBruteForceBlocker/Integrations/FeedBruteForceBlocker/FeedBruteForceBlocker.py]:
```suggestion
        "updatedate": {
```

File: Packs/FeedBruteForceBlocker/Integrations/FeedBruteForceBlocker/FeedBruteForceBlocker.yml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  version: -1
+configuration:
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
```
Comment (avidan-H) [inline Packs/FeedBruteForceBlocker/Integrations/FeedBruteForceBlocker/FeedBruteForceBlocker.yml]: You have `proxy` and `insecure` twice in the YAML.

File: Packs/FeedBruteForceBlocker/Integrations/FeedBruteForceBlocker/FeedBruteForceBlocker.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            "regex": r"(\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2})",
+            "transform": "\\1"
+        }
+    })
+    params['fields'] = fields
```
Comment (avidan-H) [inline Packs/FeedBruteForceBlocker/Integrations/FeedBruteForceBlocker/FeedBruteForceBlocker.py]: What about the `custom_whatever_it's_called` parameter that you added to the httpAPImodule? You haven't included it here.

Comment (roysagi) [reply Packs/FeedBruteForceBlocker/Integrations/FeedBruteForceBlocker/FeedBruteForceBlocker.py]: will add after it is merged.

File: Packs/FeedBruteForceBlocker/Integrations/FeedBruteForceBlocker/FeedBruteForceBlocker.yml (hunk 2 above)
Comment (roysagi) [reply Packs/FeedBruteForceBlocker/Integrations/FeedBruteForceBlocker/FeedBruteForceBlocker.yml]: fixed

File: Tests/conf.json [hunk 4]
```diff
@@ -2182,6 +2182,10 @@
         {
             "playbookID": "Indicator Feed - Test",
             "fromversion": "5.5.0"
+        },
+        {
+            "playbookID": "playbook-BruteForceBlocker_test",
```
Comment (ronykoz) [inline Tests/conf.json]: Don;t you need an integration as well?

File: Packs/FeedBruteForceBlocker/Integrations/FeedBruteForceBlocker/FeedBruteForceBlocker.yml (hunk 2 above)
Comment (ronykoz) [reply Packs/FeedBruteForceBlocker/Integrations/FeedBruteForceBlocker/FeedBruteForceBlocker.yml]: you should have kept the ones at the bottom. This is not a good place for having them

Comment (roysagi) [reply Packs/FeedBruteForceBlocker/Integrations/FeedBruteForceBlocker/FeedBruteForceBlocker.yml]: done

File: Tests/conf.json (hunk 4 above)
Comment (roysagi) [reply Tests/conf.json]: yup...

File: Packs/FeedBruteForceBlocker/Integrations/FeedBruteForceBlocker/FeedBruteForceBlocker.py (hunk 3 above)
Comment (ronykoz) [reply Packs/FeedBruteForceBlocker/Integrations/FeedBruteForceBlocker/FeedBruteForceBlocker.py]: lets add that now please, and merge the other PR

File: Packs/FeedBruteForceBlocker/Integrations/FeedBruteForceBlocker/FeedBruteForceBlocker.yml [hunk 5]
```diff
@@ -0,0 +1,97 @@
+category: Data Enrichment & Threat Intelligence
+fromversion: 5.5.0
+commonfields:
+  id: BruteForceBlocker Feed
+  version: -1
+configuration:
+- display: Fetch indicators
+  name: feed
```
Comment (avidan-H) [inline Packs/FeedBruteForceBlocker/Integrations/FeedBruteForceBlocker/FeedBruteForceBlocker.yml]:
```suggestion
  name: feed
  defaultvalue: true
```

File: Packs/FeedBruteForceBlocker/Integrations/FeedBruteForceBlocker/FeedBruteForceBlocker.py (hunk 3 above)
Comment (roysagi) [reply Packs/FeedBruteForceBlocker/Integrations/FeedBruteForceBlocker/FeedBruteForceBlocker.py]: done

**Comment by roysagi** [comment]
All field descriptions are default and @kirbles19 already approved the description in the TIP excel. So adding docs approved.

**Comment by ronykoz** [comment]
Don't merge, waiting on an image

**Pull request merged**